src/lib.rs

```diff
@@ -277,6 +322,13 @@ where
         self
     }
 
+    ///
```

Comment:
Could you please put a comment here? Explain what `id` is, and what syntax to use with `shortcut`. Also, provide a link to the docs.